### ISSUE TITLE
Convert event to value and vice-versa

### ISSUE THREAD

**Issue by dronus** [cbdevnet/midimonster#147]
When converting MIDI notes into OSC commands for example, it would be useful to know if / how to convert the event id (that is the note id in this case) into a value that can be used in the OSC packet.

For example:

    midi_in.ch0.note{1..64} > osc_out./note

would just trigger /note, but loose the information which note was played. Same goes for all value outputs, like midi CC or evdev messages. Same, the other way round, goes for playing MIDI notes with mouse coordinates for example.

Of course, the notes may be converted to different OSC paths with ease, like 

    midi_in.ch0.note{1..64} > osc_out./note{1..64}

but that would require the receiving device to read the note pitch from the _path_ instead of an integer note number _value_, as it is sometimes the case.

So basically, translating event path wildcard ids into values and vice-versa would be useful.
